core/src/test/java/com/oath/oak/UnsafeUtilsTest.java

```diff
@@ -12,134 +12,6 @@
 
 public class UnsafeUtilsTest {
 
-
-    @Test
-    public void testUnsafeCopy() {
```

Comment:
strange that we have such high test coverage for a piece of code that is actually redundant ?

Reply:
It was used in Druid, but now we realized that such unsafe behavior should be implemented by the user if needed and not be given as a service.